src/pages/observability-packs.js

```diff
@@ -202,60 +212,117 @@ const ObservabilityPacksPage = ({ data, location }) => {
             }
           `}
         >
-          <FormControl>
-            <Label htmlFor="sortFilter">Sort by</Label>
-            <Select
-              id="sortFilter"
-              value={sortState}
-              onChange={(e) => {
-                setSortState(e.target.value);
-                document.getElementById(e.target.id).blur();
-                tessen.track('observabilityPack', `packSort`, {
-                  packSortState: e.target.value,
-                });
+          <div
+            css={css`
+              input {
+                background: inherit;
+              }
+            `}
+            style={{
+              width: `${searchExpanded ? '300px' : '50px'}`,
+              transition: 'all 0.5s ease',
+            }}
+          >
+            {!searchExpanded && (
+              <Button
+                variant={Button.VARIANT.PLAIN}
+                css={css`
+                  background: none;
+                  border: none;
+                  color: inherit;
+                  cursor: pointer;
```

Comment:
I don't think we need the following styles:
* [`background: none;`](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/Button.js#L69)
* [`color: inherit;`](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/Button.js#L68) (I think)
* [`cursor: pointer;`](https://github.com/newrelic/gatsby-theme-newrelic/blob/develop/packages/gatsby-theme-newrelic/src/components/Button.js#L121)
```suggestion
                  border: none;
```

Reply:
ah yep. leftover from previous non-button implementation. thanks!